### ISSUE TITLE
add removeBeneficiaries

### ISSUE THREAD

**Issue by obernardovieira** [impactMarket/utils#205]
[removeBeneficiaries](https://github.com/impactMarket/impact-market-smart-contracts/blob/main/contracts/community/CommunityImplementation.sol#L1005)

limit to an array of 20

**Comment by Marianavpt** [comment]
Is this issue solved @PauloSousapt ?

**Comment by obernardovieira** [comment]
> Is this issue solved @PauloSousapt ?

No. And currently not in the pipeline. This is an improvement over a mechanism that already exists.
